asana_tasks/triangular_treasure/triangular_treasure.rb

```diff
@@ -11,6 +11,7 @@
 #   triangular(3)==6,
 #   triangular(-10)==0
 
+
 def triangular( n )
   return 0 if n <= 0
   return n*n - triangular(n - 1)
```

Comment:
@Alexander-Andrade Just for the education purposes, please change the function to have ternary operator and make one more commit to this branch. Let me know when it is done, so I will check pull request again.